staging/src/k8s.io/cloud-provider/volume/helpers/zones.go

```diff
@@ -135,7 +138,7 @@ func SelectZonesForVolume(zoneParameterPresent, zonesParameterPresent bool, zone
 	}
 
 	if (len(allowedTopologies) > 0) && (allowedZones.Len() == 0) {
-		return nil, fmt.Errorf("no matchLabelExpressions with %s key found in allowedTopologies. Please specify matchLabelExpressions with %s key", v1.LabelFailureDomainBetaZone, v1.LabelFailureDomainBetaZone)
+		return nil, fmt.Errorf("no matchLabelExpressions with %s key found in allowedTopologies. Please specify matchLabelExpressions with %s key", v1.LabelTopologyZone, v1.LabelTopologyZone)
```

Comment:
> Is this logic shared by other clouds too? Will this cause other clouds' Provision() to use GA labels?

This function `SelectZoneForVolume` is shared by all in-tree plugins and is being called in in-tree code. This function only picks a zone value from the labels. So it does not matter if the value comes from GA or beta label. For example:

If a node has `failure-domain.beta.kubernetes.io/zone=us-west-2` and `topology.kubernetes.io/zone=us-west-2`. This function will just return the `us-west-2` so it should not matter here.